pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py

```diff
@@ -57,6 +58,8 @@ def __init__(
         cmd = [wokwi_cli, '--interactive', app.app_path]
         if (wokwi_timeout is not None) and (wokwi_timeout > 0):
             cmd.extend(['--timeout', str(wokwi_timeout)])
+        if (wokwi_scenario is not None) and os.path.exists(wokwi_scenario):
+            cmd.extend(['--scenario', wokwi_scenario])
```

Comment:
Not sure about it, when enabling `--scenario ...`, we should not use `--interactive`, right?

Reply:
`--interactive - Redirect stdin to the simulated serial port`
I run multiple tests as it is with the `--interactive` set, and all worked well. Pytest with expecting some output or with UNITY inside the sketch.

Reply:
cool. no problem then.